code/game/objects/ovipositor.dm

```diff
@@ -1,98 +1,38 @@
-#define QUEEN_OVIPOSITOR_DECAY_TIME 500
+#define QUEEN_OVIPOSITOR_DECAY_TIME 10 MINUTES
```

Comment:
whoa stealth nerf?

Reply:
500 isn't nice in minutes

Reply:
Talked offline about this, was originally 50 seconds but happy to have it at 10 minutes because it allows the world to keep some alien corruption as xenos move about.